index.js

```diff
@@ -1,72 +1,25 @@
-class Book {
-  constructor(title, author) {
-    this.title = title;
-    this.author = author;
-    this.books = [];
-  }
-
-  getBooks() {
-    let savedBooks = [];
-    if (localStorage.getItem('books')) {
-      savedBooks = JSON.parse(localStorage.getItem('books'));
-    }
-    this.books = savedBooks;
-    return savedBooks;
-  }
-
-  showBooks() {
-    this.getBooks();
-    const bookList = document.querySelector('#book-list');
-    bookList.innerHTML = '';
-    this.books.forEach((book, index) => {
-      const row = document.createElement('tr');
-      row.classList.add('w-100');
-      row.innerHTML = `
-        <td>${book.title}</td>
-        <td>${book.author}</td>
-        <td><button data-index="${index}" class="btn-btn-danger delete">Delete</button></td>
-      `;
-      bookList.appendChild(row);
-    });
-
-    const rBook = document.querySelectorAll('.delete');
-
-    rBook.forEach((remove) => {
-      remove.addEventListener('click', this.removeBook);
-    });
-  }
-
-  addBook(book) {
-    this.getBooks();
-    this.books.push(book);
-    localStorage.setItem('books', JSON.stringify(this.books));
-    this.showBooks();
-  }
-
-  removeBook = (event) => {
-    const { index } = event.target.dataset;
-    this.getBooks();
-    this.books = this.books.filter((book, i) => i !== Number(index));
-    localStorage.setItem('books', JSON.stringify(this.books));
-    this.showBooks();
-  };
-}
-
+// eslint-disable-next-line import/extensions
+import { DateTime } from './node_modules/luxon/src/luxon.js';
+// eslint-disable-next-line import/extensions
+import Book from './book.js';
+// eslint-disable-next-line import/extensions
+import Library from './library.js';
+
+const booksSection = document.querySelector('.books');
 const form = document.querySelector('#form');
 const title = document.querySelector('#book-title');
 const author = document.querySelector('#author');
 
-const book = new Book();
-
-book.showBooks();
+const library = new Library(booksSection);
+library.books = library.getBooks();
+library.showBooks();
```

Comment:
Excellent job refactoring your code to use `es6` functionalities👍. currently, your navigation links are not working and it is impossible to display the other pages of the application🥲. I suggest you kindly fix that👍.


https://user-images.githubusercontent.com/64643995/236780139-bf3fec41-2110-4d65-9a78-0ec9c1cd74b1.mp4

Reply:
Noted with thanks

Reply:
/home/chrisdroid/Videos/Screencasts/Screencast from 08-05-2023 12:45:13 ALASIRI.webm

I have not changed anything but the code works as expected. Kindly assist.